src/main/java/eu/dissco/orchestration/backend/domain/ExportType.java

```diff
@@ -0,0 +1,24 @@
+package eu.dissco.orchestration.backend.domain;
+
+import lombok.Getter;
+
+@Getter
+public enum ExportType {
+  DWC_DP("dwc-dp"),
+  DWCA("dwca");
+
+  private final String urlName;
+
+  ExportType(String urlName) {
+    this.urlName = urlName;
+  }
+
+  public static ExportType fromName(String name) {
```

Comment:
If we get many enum types it might be nice to store the name and enum in a map instead, but this is definitely enough for now